src/app/features/public-form/edit-publicform-route/edit-publicform-route.component.html

```diff
@@ -15,7 +15,8 @@
     <span
       i18n
       matTextPrefix
-      [ngClass]="{ 'prefix-text': !formControl.disabled }"
+      class="prefix-text"
+      [ngClass]="{ 'prefix-text-disabled': !formControl.disabled }"
```

Comment:
Now it's too wide :see_no_evil: 
![image](https://github.com/user-attachments/assets/bb51cfc2-db7c-440c-8b7b-957709fd12a3)

Reply:
fixed this please have a look now

Reply:
this can still be wider than its parent:

![image](https://github.com/user-attachments/assets/d3454dce-46f9-4f31-96d7-650b82a9a366)


Please test it thoroughly with a continuous change of screen widths